examples/causal_inference/interventional_distribution.myst.md

```diff
@@ -0,0 +1,682 @@
+---
+jupytext:
+  notebook_metadata_filter: substitutions
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc_env
+  language: python
+  name: pymc_env
+---
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "tags": []}
+
+(interventional_distribution)=
+# Interventional distributions and graph mutilation with the do-operator
+
+:::{post} July, 2023
+:tags: causal inference, do-operator, graph mutilation
+:category: beginner, explanation
+:author: Benjamin T. Vincent
+:::
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "tags": []}
+
+:::{attention}
+This notebook relies on experimental functionality currently in the [pymc-experimental](https://github.com/pymc-devs/pymc-experimental) repository. In the near future this will be moved into the main [pymc](https://github.com/pymc-devs/pymc) repository.
+:::
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "tags": []}
+
+[PyMC](https://github.com/pymc-devs/pymc) is a pivotal component of the open source Bayesian statistics ecosystem. It helps solve real problems across a wide range of industries and academic research areas every day. And it has gained this level of utility by being accessible, powerful, and practically useful at solving _Bayesian statistical inference_ problems.
+
+But times are changing. There's a [causal revolution](https://en.wikipedia.org/wiki/The_Book_of_Why) underway and there's a growing recognition that to answer some of the most interesting and challenging questions requires us to intergrate causal reasoning into our efforts.
+
+PyMC is rising to this challenge! While there are many novel causal concepts to learn, Bayesians will find that they are not starting from scratch. They are already pretty familiar with [Directed Acyclic Graphs (DAGs)](https://en.wikipedia.org/wiki/Directed_acyclic_graph) and so this gives a good jumping off point to gain relatively easy access into the world of **Bayesian causal inference**.
+
+This notebook is going to cover one of the foundational components of causal reasoning which has newly arrived to the PyMC ecosystem, the $\operatorname{do}$ operator. Indeed, depending on whose definitions you want to use, adding the $\operatorname{do}$ operator into the kind of Bayesian DAGs that PyMC users build every day gets us to the status of building [causal Bayesian networks](https://en.wikipedia.org/wiki/Causal_graph). 
+
+If that sounds cool, let's dive in...
+
++++
+
+## Set up the notebook
+
```

Comment:
we should also add daft here following https://www.pymc.io/projects/docs/en/latest/contributing/jupyter_style.html#extra-dependencies, not sure about pymc-experimental

Reply:
I've given that a go